### DISCUSSION TITLE
Using v3 studio inside a NextJS app?

### DISCUSSION THREAD

**Discussion by hoggworks** [sanity-io/sanity#3622] [Sanity Studio Support] [answered]
Hey all! I was watching this video about using studio in v3 within an application: https://youtu.be/298mlqa1-Hk And I'm wondering if there's any documentation on how to do that yet? 

Thanks!

**Comment by unitof** [comment, marked as answer]
Hot and fresh! https://github.com/sanity-io/sanity/discussions/3479

**Comment by loremipson** [comment]
I have no publish button when embedding into Next.

**Reply by loremipson** [reply]
Downgrading react 18 to 17 seems to fix this issue.

edit: #3604 has already reported this.
